.buildkite/nightly-benchmarks/run-benchmarks-suite.sh

```diff
@@ -70,24 +70,11 @@ wait_for_server() {
 
 kill_gpu_processes() {
   # kill all processes on GPU.
-  pids=$(nvidia-smi --query-compute-apps=pid --format=csv,noheader)
-  if [ -z "$pids" ]; then
-      echo "No GPU processes found."
-  else
-      for pid in $pids; do
-          kill -9 "$pid"
-          echo "Killed process with PID: $pid"
-      done
 
-      echo "All GPU processes have been killed."
-  fi
+  pkill -f python3
+  ps -e | grep pt_main_thread | awk '{print $1}' | xargs kill -9
```

Comment:
This was removed by @cadedaniel

Reply:
the `pkill -f python3` makes it hard to run this locally when other python processes live. happy to help craft a better command.

Reply:
optionally, can have a flag to skip this kill if provided. im happy to make a PR for that

Reply:
Currently the command `pkill-f python3` is mainly used to kill the openAI server -- it is now in a separate process. Would be great if @cadedaniel you know a graceful way to shutdown the openAI server.

Reply:
`ps aux | grep python | grep openai | awk '{print $2}' | xargs -r kill -9`